snap/snapcraft.yaml

```diff
@@ -10,12 +11,19 @@ description:        |
   convenient scrolling.
 
 grade:              stable
-confinement:        strict
+confinement:        devmode
```

Comment:
Change the confinement is a side effect?

Reply:
I was testing it out, and forgot to change it back to strict

Reply:
Can you revert this on the branch?